### DISCUSSION TITLE
docs?

### DISCUSSION THREAD

**Discussion by bdelamatre** [rappasoft/laravel-livewire-tables#237] [General]
What were you thinking for expanded docs? Trying to do something inside Github like the Wiki or an external tool?

**Comment by rappasoft** [comment]
I'm not sure what all the cool kids use these days. There are so many options. I usually just stick to readme or a custom site but this one is getting really long and I don't feel like making a custom site.

**Comment by bdelamatre** [comment]
what about this?

https://docs.github.com/en/communities/documenting-your-project-with-wikis/about-wikis

**Reply by rappasoft** [reply]
I'm sure that will more than suffice, I'll look into it tonight.

**Reply by rappasoft** [reply]
I've started here, github wiki should be plenty: https://github.com/rappasoft/laravel-livewire-tables/wiki

**Comment by bdelamatre** [comment]
Nice start.  Do you want to open permissions to me so I can help out a bit?

https://docs.github.com/en/communities/documenting-your-project-with-wikis/changing-access-permissions-for-wikis

**Reply by rappasoft** [reply]
I was looking at that but there doesn't seem like a way to give only you access to edit, it's like everyone or people will full access to the whole repo. Unless I'm reading it wrong.

**Reply by bdelamatre** [reply]
I haven't done it before, but the docs imply your should be able to:

"Only repository collaborators can edit a public repository's wiki by default, but you can allow anyone with a GitHub account to edit your wiki."

**Reply by rappasoft** [reply]
It's checked right now, I think by collaborators it means access to the entire repository. GitHub doesn't have permissions unfortunately unless you run an organization I think

**Comment by bdelamatre** [comment]
I might consider re-adding the simple example back to the readme and making the new documentation link more obvious.

**Reply by rappasoft** [reply]
Done!

**Reply by camya** [reply]
Hi Anthony, maybe you can also re-add **screenshots for both the basic and the advanced example** to the **README.md**. 

This gives new developers an instant idea, what your great package can do for them. ;)
